src/base.jl

```diff
@@ -133,6 +167,10 @@ struct PowerSystem{L <: ElectricLoad,
 
 end
 
+# DOCTODO JJS What is the purpose of this statement? OK, it looks like a
```

Comment:
Correct, this is for missing fields and also to enable keyword arguments in the constructors